readthedocs/api/v3/serializers.py

```diff
@@ -526,7 +526,7 @@ class SubprojectCreateSerializer(FlexFieldsModelSerializer):
 
     child = serializers.SlugRelatedField(
         slug_field='slug',
-        queryset=Project.objects.all(),
+        queryset=Project.objects.none(),
```

Comment:
Can't this just be omitted?

Reply:
`SlugRelatedField` requires queryset as an `argument`.